### DISCUSSION TITLE
getZoomLevel doesn't appear to be exported

### DISCUSSION THREAD

**Discussion by kpierro-cb** [chartjs/chartjs-plugin-zoom#858] [General]
When I try to use getZoomLevel, webpack gives me a compile error that it's not available:

`export 'getZoomLevel' (imported as 'getZoomLevel') was not found in 'chartjs-plugin-zoom' (possible exports: default, pan, resetZoom, zoom, zoomRect, zoomScale)`

**Comment by bavoco** [comment]
Try `chart.getZoomLevel()`
